tools/diff-processor/rules/breaking_changes_test.go

```diff
@@ -332,7 +332,7 @@ func TestComputeBreakingChanges(t *testing.T) {
 			schemaDiff := diff.ComputeSchemaDiff(tc.oldResourceMap, tc.newResourceMap)
 			violations := ComputeBreakingChanges(schemaDiff)
```

Comment:
Could you add tests here that the standard fields (resource, field, documentationreference, rule*) are set as expected? (This would allow us to refactor the internals later and easily verify the behavior hasn't changed.)

Reply:
Done.